### ISSUE TITLE
train subcommand seems to sample only a subset of my dataset

### ISSUE THREAD

**Issue by basile-parmelli** [JacksonBurns/fastprop#7]
Hello, 

First of all, thank you for releasing fastprop !
I am a chemistry student (MSc) and just getting into cheminformatics for a project. As I am a total beginner my issue might be very simple to fix. 

I installed fastprop by cloning the repo and used it through the Powershell terminal, in order to train the model (and get metrics values) on a dataset I extracted. Out of 2000+ entries (with duplicates), descriptors were only calculated for 400 molecules (according to the progress bar).

I curated the dataset (got rid of duplicates) and the input csv file is now about 700 entries long, but when I call the `train` subcommand, the progress bar for descriptors calculation only mentions "x/190 done". 

Nonetheless, the log mentions `"INFO :  Size after preprocessing: (760, 730)"` which hints that the number of entries was preserved. This leads me to think that maybe I do not understand what is actually computed regarding descriptors. 

I tried being specific with arguments, explicitly adding `-ns 0.8 -vs 0.1 -ts 0.1` after the `train `subcommand for instance, but it doesn't seem to change much.

Long story short, could you explain why it appears that descriptors are only computed for about 25% of the total dataset ? Did I do something wrong ?

Thanks in advance for any answer, and feel free to ask for more info if needed.

**Comment by JacksonBurns** [comment]
hi @basile-parmelli sorry I missed this issue when you opened it - can you look at the cached descriptor file to see if it is showing the full number of rows in the dataset?
